packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-success-dialog.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Button from '../button';
+import Modal from '../modal';
+import Text from '../text';
+import WalletCFDCard from './wallet-cfd-card';
+import { isMobile } from '@deriv/shared';
+import './wallet-cfd-success-dialog.scss';
+
+export type TWalletCFDCard = {
+    account_title: string;
+    app_icon: string;
+    balance: string;
+    currency_title: React.ReactNode;
+    currency: string;
+    gradient_header_class: string;
+    icon: string;
+    is_demo: boolean;
+    wallet_label: string;
+};
+
+type TWalletCFDSuccessDialog = {
+    header: string;
+    is_demo?: boolean;
+    is_open: boolean;
+    message: React.ReactNode;
+    onSubmit?: () => void;
+    submit_button_text?: string;
+    toggleModal?: () => void;
+    wallet: TWalletCFDCard;
+};
+
+const WalletCFDSuccessDialog = ({
+    header,
+    is_open,
+    message,
+    onSubmit,
+    submit_button_text,
```

Comment:
@aizad-deriv you deleted `{submit_button_text || localize('OK')}` in the code next, in this case would be better to write default value if it's undefined
```suggestion
    submit_button_text = 'OK',
```